src/types.ts

```diff
@@ -47,3 +49,33 @@ export type DocumentInternationalizationMenuProps = {
   schemaType: ObjectSchemaType
   documentId: string
 }
+
+// Extend Sanity schema definitions
+export interface DocumentInternationalizationSchemaOpts {
+  documentInternationalization?: {
+    /** Set to true to disable duplication of this field or type */
+    exclude?: boolean
+  }
+}
+
+declare module 'sanity' {
```

Comment:
@bjoerge @rexxars - any better pattern for how to do this in maintainable way?

Reply:
right, but wouldn't it be easier if all the `*Options` interfaces had a common parent that you could extend from instead of each one individually?

Reply:
Since it's only a subset of options that should be extended, we'd then have export (and maintain!) a separate union type of only the options that should be extended here, so I think this is preferable. We hardly ever add new schema types, so don't think it will be a problem in practice either.